public/js/directives/years/years-directive.html

```diff
@@ -26,16 +31,19 @@ <h3 ng-if="::readonly">{{::year.title}}</h3>
                         </li>
                         <button ng-if="::!readonly" class="btn btn-default add-class-btn" data-as-sortable-item
                             ng-click="addCourse(semester)">
-                            <i class="fa fa-plus" aria-hidden="true" margin-top="5px"></i>
+                            <i class="fa fa-plus" aria-hidden="true"></i>
                         </button>
+                        <div ng-if="::!readonly" class="delete-semester-btn"
+                                ng-click="removeSemester(year,$index)">
+                            <i class="fa fa-times" aria-hidden="true"></i>
+                        </div>
                     </ul>
                 </div>
-                <button ng-if="::!readonly" class="btn btn-default btn-xs" ng-click="removeSemester(year,$index)">Delete</button>
             </div>
         </div>
         </li>
     </ol>
 
     <button ng-if="::!readonly" class="btn btn-default add-year-btn"
-        ng-click="addYear()" style="margin-left:15px">Add Year</button>
+        ng-click="addYear()" style="margin-left:15px"><i class="fa fa-plus" aria-hidden="true"></i>&nbsp;&nbsp;Add Year</button>
```

Comment:
Toy with the fa-fw class. No idea what it does but sometimes it makes things work better for me. Also, I think actual spaces work too. This is fine though, up to you if you wanna change it